webapp/src/components/Translation.vue

```diff
@@ -15,6 +18,7 @@
     <div
       v-else
       class="wrapper"
+      v-if='noTranslation === false'
```

Comment:
I need to update this to v-else-if, but works as is. Will update after merging into development.

Reply:
I recommend adding a TODO comment or github issue so you don't forget. I would forget. :-)

Reply:
https://github.com/awslabs/aws-media-insights-engine/issues/83